### PR TITLE
Data Foundation tab should not be blocked on StorageCluster presence

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
**Before:**
Screenshot from 2022-08-02 21-10-38

**After:**
Screenshot from 2022-08-02 21-11-28

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
       "href": "/odf/cluster"
     },
     "flags": {
-      "required": ["ODF_MODEL", "OCS"],
+      "required": ["ODF_MODEL"],
```
Comment (bipuladh) [inline console-extensions.json:151]: We need to fix the issue around unprivileged user as well. Apparently this fix is not enough.

Comment (SanjalKatiyar) [reply console-extensions.json:151]: yeah but that is a different bug. Do we want to fix 2 different bugs in same PR ?

Comment (SanjalKatiyar) [reply console-extensions.json:151]: this PR was for solving regression... but if u want I can add fix for other bug in same PR as well...

**Pull request merged**